SUMMARY.md

```diff
@@ -13,7 +13,8 @@ SPDX-License-Identifier: CC-BY-4.0
 * [Functional Architecture](FUNCTIONAL_ARCHITECTURE.md)
     * [SSD import from CIM](CIM_61850_MAPPING.md)
     * [SSD import from CIM - MVP](CIM_61850_MAPPING_MVP.md)
-* [Technical Architecture](TECHNICAL_ARCHITECTURE.md)
+* [Technical Architecture](technical/TECHNICAL_ARCHITECTURE.md)
+  * [CoMPAS SCL Data Service](technical/COMPAS_SCL_DATA_SERVICE.md)
```

Comment:
Structure is fine, but now we have 2 separate folders containing images (one specific for the technical stuff, one globally). Maybe good to define a final structure.